example/example.ino

```diff
@@ -12,11 +12,11 @@ void loop() {
             a[i][0] = random(100);
             a[i][1] = i;
         }
-  
+
         //  sorter::sort_max(a); // Set two first indexes in array to max values
-        sorter::sort_min(a); // Set two first indexes in array to min values
-        //  sorter::swap(); // Swaps two numbers 
-        
+        utils::sort_min(a); // Set two first indexes in array to min values
```

Comment:
That would be excellent! But be careful because the alignment will change when the comment signs in front of actual function calls are removed. (So probably uncomment those lines and align like that. It's an example after all, doesn't have to run :-D)